### ISSUE TITLE
Start animation after page is loaded

### ISSUE THREAD

**Issue by erpixxx** [ExvoidNet/site#9]
Currently the animation starts immediately when the user enters the page, which causes some visual bugs. We should create some JS script that employs class manipulation, activating the animation only upon complete page load to ensure a smoother. However, there might other solutions worth considering.

**Comment by Matou0014** [comment]
I agree + I think we should add some delays to some of the animations.
